src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -631,6 +631,15 @@ public LibraryTab getCurrentLibraryTab() {
         return (LibraryTab) tabbedPane.getSelectionModel().getSelectedItem();
     }
 
+    public void setCurrentTab(BibDatabaseContext databaseContext) {
```

Comment:
An architectural issue: You want to select a tab by providing a database context. I think the level of abstraction is broken here.